pkg/diagnostics/v2/client.go

```diff
@@ -76,6 +79,35 @@ func (c *Client) Download(id string, dst io.Writer) error {
 	}
 }
 
+// Create creates a new cluster bundle and returns its ID.
+func (c *Client) Create() (string, error) {
+	req, err := c.http.NewRequest("PUT", fmt.Sprintf("%s/%s", baseURL, uuid.NewV4().String()), nil)
```

Comment:
nit: `.String()` shouldn't be necessary here

Reply:
Yes sure. I like explicit code, though, but I have no clue whether there's a beat practice for this case.